src/Comms.cpp

```diff
@@ -22,9 +38,15 @@ namespace Comms {
     }
 
     void emitPacket(Packet *packet) {
-        Serial.print(packet->data[0]);
-        Serial.print(packet->data[1]);
-        Serial.print(packet->data[2]);
-        Serial.println(packet->data[3]);
+        //Send over serial
+        for (int i = 0; i < packet->len; i++) {
+            Serial.println(packet->data[i]);
```

Comment:
Use Serial.write instead, since it writes each byte directly to the UART port without any other processing